Sources/Transaction.swift

```diff
@@ -10,13 +10,40 @@ import BigInt
 public struct Transaction {
     public var accountNonce: UInt64
```

Comment:
can we call it `nonce` for consistency?

Reply:
Again, based on the geth implementation.

Reply:
I guess the JSON names do match what you suggest, I'll change it.